dashboard/src/index.html

```diff
@@ -8,12 +8,14 @@
 
   <link href="img/favicon.ico" rel="shortcut icon" type="image/vnd.microsoft.icon" />
   <!-- STYLES -->
-  <link rel="stylesheet" type="text/css" href="css/index.css">
   <!-- build:css lib/css/main.min.css -->
   <link rel="stylesheet" type="text/css" href="components/bootstrap/dist/css/bootstrap.min.css">
   <link rel="stylesheet" type="text/css" href="components/font-awesome/css/font-awesome.min.css">
   <link rel="stylesheet" type="text/css" href="components/rdash-ui/dist/css/rdash.min.css">
+  <!-- Load our custom CSS last so it overwrites the minified CSS above -->
+  <link rel="stylesheet" type="text/css" href="components/css/index.min.css">
```

Comment:
Hmmm but that won't work, because that stylesheet link refers to the build folder, when `gulp build` is run, then the build folder is generated and so is the css -> index.css sub-folder.

```
build ->
           -- css
           -- css

           -- css

```